org.jacoco.doc/docroot/doc/build.html

```diff
@@ -267,18 +267,22 @@ <h2>Compilation and testing with different JDKs</h2>
   <li><code>mvn clean verify -Djdk.version=6 -Dbytecode.version=6</code></li>
   <li><code>mvn clean verify -Djdk.version=7 -Dbytecode.version=7</code></li>
   <li><code>mvn clean verify -Djdk.version=8 -Dbytecode.version=8</code></li>
-  <li><code>mvn clean verify -Djdk.version=8 -Dbytecode.version=8 -Decj</code></li>
+  <li><code>mvn clean verify -Djdk.version=8 -Dbytecode.version=8</code></li>
```

Comment:
I think the whole line should be deleted now.

Reply:
Indeed, don't know why missed this 😆 done